### DISCUSSION TITLE
Welcome to podcast-namespace Discussions!

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#362] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adamc199** [comment]
Suggest we change this from Podcast namespace discussion to Podcast Standard Working Group.

**Comment by theDanielJLewis** [comment]
I like that @adamc199 didn’t use the word “the,” which often confuses branding, IMO.

**Comment by DavidBucienski** [comment]
I need clarification on the tag <podcast:locked>

The official description:
"This tag may be set to yes or no. The purpose is to tell other podcast platforms whether they are allowed to import this feed."

What qualifies as "podcast platforms", I ask, because I remember Adam saying that some people have Patreon only content. They want to keep it exclusive. But if a podcast player respected this, then shows like No Agenda and Moe Facts would not be found by users, because they are locked. 

Specifically, is a podcast player a "podcast platform"?

**Comment by PofMagicfingers** [comment]
You're right, it's not the right term. Podcast platform, here should be
"public podcast directory". Like iTunes store, Spotify, searches inside
apps or such.

It's meant to declare your choice of public indexation in a podcast app
service.


I need clarification on the tag podcast:locked

**Reply by DavidBucienski** [reply]
So you're saying it should read?

"This tag may be set to yes or no. The purpose is to tell search engines other than Podcast Index, whether they are allowed to index this feed."

**Reply by daveajones** [reply]
No the locked tag is not a crawler control. The block tag will be for that. The locked tag tells platforms whether they are allowed to import the feed or not.

**Reply by DavidBucienski** [reply]
This takes me and @PofMagicfingers full circle, as to what a platform is. It can't be a podcast player app because that would mean The No Agenda Show, Mo Facts and others should be hidden from searches. These shows are locked with a yes value.

**Reply by podcastindex** [reply]
It's not about indexing or displaying.  It's about "importing".  A platform, in this context, is a hosting platform (Anchor, RSS.com, Buzzsprout, Blubrry, etc.).  Any service that would *host* the feed.

**Reply by DavidBucienski** [reply]
@daveajones Thanks for the clarification. So the description for podcast:locked should read:

"This tag may be set to yes or no. The purpose is to tell other podcast _hosting_ platforms whether they are allowed to import this feed."

**Reply by PofMagicfingers** [reply]
You're right, I read the question too fast, and I mixed up `podcast:locked` and `podcast:block`. `podcast:locked` is about allowing or not import from other hosts. `podcast:block` is equivalent to `itunes:block` but platform/directory independant.

**Reply by daveajones** [reply]
> @daveajones Thanks for the clarification. So the description for podcast:locked should read:
> 
> "This tag may be set to yes or no. The purpose is to tell other podcast _hosting_ platforms whether they are allowed to import this feed."

Good call.  Fixed:  [Commit](https://github.com/Podcastindex-org/podcast-namespace/commit/e2501e5f6776253f839590017555eb3039168649)

**Reply by theDanielJLewis** [reply]
I suggest changing "platforms" to "providers" to further help distinguish the meaning, and a hyphen to be grammatically correct. And even a little more to clarify it further.

> This tag may be set to yes or no. The purpose is to tell other podcast-hosting providers whether they are allowed to migrate/import this feed to a podcaster's account.

**Comment by DavidBucienski** [comment]
A google search for either term gets you the same result. One of the results is an article using the term platform. Seems like platform is the common vernacular.

**Reply by theDanielJLewis** [reply]
Sure, "platform" is common, but I think it's a bit ambiguous, as this conversation demonstrates. And thus why I think we should use more accurate language to avoid confusion.

**Reply by DavidBucienski** [reply]
This conversation started when the term "hosting" was not used. Since there are multiple platforms (podcast players, indexers and hosting) it was unclear which group the locked tag was applicable to. Adding the term "hosting" clarified this.

Since "providers" and "platforms" are used interchangeably in this industry, either could be used.
